specs/pvs/corla.pvs

```diff
@@ -1043,14 +1098,18 @@ END cast_vote_records
 
 %+BEGIN ui
 
-%%% \section{User Interfaces}
+%%% \subsection{User Interfaces}
 
 %%% The user interfaces of the system are the visible interactive
 %%% parts of the application. There are three different user
 %%% interfaces in the RLA Tool. The precise names of these interfaces
 %%% are still under discussion; one is for CDOS personnel responsible
 %%% for the audit at the state-level, one is for county personnel at
 %%% the county-level, and one is for audit board members.
+%%% Note that all these user interfaces both receive and 
+%%% provide information.
+%%% Because the public dashboard only pushes information out, 
+%%% we do not consider it a "user interface".
```

Comment:
I think we ought to add the public dashboard to the list.  It public dashboard will not change the database, but will accept some sorts of interaction and/or input, whether mouse clicks or searches or the like.